developer/src/kmcmplib/subprojects/.gitignore

```
@@ -2,3 +2,4 @@
 /*.zip
 /*.tgz
 /packagecache
+/googletest-1.14.0
```

```suggestion
/googletest-*
```

but i wonder if the subdir needs to be versioned?

---

I think we can generalise the .gitignore, to save tracking the version if we upgrade later. Done